src/Yandex/Allure/Adapter/Annotation/TestId.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Yandex\Allure\Adapter\Annotation;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+
+/**
+ * @Annotation
+ * @Target({"CLASS", "METHOD"})
+ * @package Yandex\Allure\Adapter\Annotation
+ */
+class TestId
```

Comment:
All is good. Let's do a small modification. To have uniform annotation names let's call it TestCaseId because respective Java annotation has such name: https://github.com/allure-framework/allure-core/blob/master/allure-java-annotations/src/main/java/ru/yandex/qatools/allure/annotations/TestCaseId.java.